gitpod-io/website
# Update cost calculators to 16 CPU Machines

## Description
This PR changes the cost calculators linked on the single cluster reference architecture to use 16 core machines instead of 8 cores - this is in line with changes to the ref archs. 

## Related Issue(s)

Fixes #

## How to test
Open preview, navigate to single cluster reference architecture, then check the linked cost calculators (AWS and GCP)

## Release Notes

```release-note
NONE
```

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2848"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2848"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
